addons/website/views/snippets/s_unveil.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_unveil" name="Unveil">
+    <section class="s_unveil pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="11">
```

Comment:
`data-row-count` doesn't match the actual grid size you are using, it should be 14 i think ?